cachix/src/Cachix/Client/Commands.hs

```diff
@@ -78,21 +80,21 @@ Continue by creating a binary cache with:
 and share it with others over https://<name>.cachix.org
   |] :: Text)
 
-create :: ClientEnv -> Maybe Config -> Text -> IO ()
-create _ Nothing _ = throwIO $ NoConfig "start with: $ cachix authtoken <token>"
-create env (Just config@Config{..}) name = do
+create :: Env -> Text -> IO ()
+create (Env Nothing _ _) _ = throwIO $ NoConfig "start with: $ cachix authtoken <token>"
```

Comment:
This should be `Env { config = Nothing }`. Imho we should avoid positional matching except in newtypes and single-field sums.

Reply:
Ah, I didn't know you can pattern match like that! Thanks